client/js/components/PhotoCarousel/PhotoCarousel.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react";
+import { View, ScrollView, Image } from "react-native";
+import styles from "./styles";
+
+class Carousel extends Component {
+  render() {
+    const { images } = this.props;
+    if (images && images.length) {
+      return (
+        <View style={styles.scrollContainer}>
+          <ScrollView
+            horizontal
+            pagingEnabled
+            showsHorizontalScrollIndicator={false}
+          >
+            {images.map((image, index) => (
+              <Image key={index} style={styles.image} source={image.source} />
+            ))}
+          </ScrollView>
+        </View>
+      );
+    }
+    console.log("Please provide images");
+    return null;
+  }
+}
+
+export default class PhotoCarousel extends Component {
+  render() {
+    const images = [
```

Comment:
put this const outside render if possible. If there is a huge array in a render, every time you render you are creating a new const with the same information.